articles/overview/overview.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Introduction to quantum computing
+description: Learn what quantum computing is, what quantum computers can do, and how you can learn quantum computing.
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.introduction
+---
+
+# Introduction to quantum computing and the Quantum Development Kit
+
+The Microsoft Quantum Development Kit (QDK) is a set of open-source tools designed to help developers learn quantum algorithms and write quantum programs. Quantum computing holds the promise to solve some of our planet's biggest challenges - in the areas of environment, agriculture, health, energy, climate, materials science, and others we haven't encountered yet.  
+
+For some of these problems, even our most powerful computers run into problems. While quantum technology is just beginning to impact the computing world, it could be far-reaching and change the way we think about computing.
+
+## What is quantum computing?
+
+In modern usage, the word quantum means the smallest possible discrete unit of any physical property, usually referring to properties of atomic or sub-atomic particles. Quantum computers use actual quantum particles, artificial atoms, or collective properties of quantum particles as processing units, and are large, complex, and expensive devices.
+
+Harnessing the unique behavior of quantum physics and applying it to computing, quantum computers introduce new concepts to traditional programming methods, making use of quantum physics behaviors such as superposition, entanglement, and quantum interference.
+
+## What can a quantum computer do?
+
+A quantum computer isn't a supercomputer that can do everything faster: it just does some select things exceptionally well.
+
+### Quantum simulation
+
+Since quantum computers use quantum phenomena in computation, they are well-suited for modeling other quantum systems. Photosynthesis, superconductivity, and complex molecular formations are examples of quantum mechanisms that quantum programs can simulate.
+
+### Cryptography and Shor’s algorithm
+
+In 1994, Peter Shor showed that a scalable quantum computer could break every widely used encryption technique. Classical cryptography relies on the intractability of operations on large numbers, such as factorization of large numbers into two prime numbers, and quantum computers excel at factorization.
+
+### Search and Grover’s algorithm
+
+In 1996, Lov Grover developed a quantum algorithm that dramatically sped up the solution to unstructured data searches, running the search in fewer steps than any classical algorithm could. This increased speed becomes significant in lists that contain millions or billions of items.
```

Comment:
```suggestion
In 1996, Lov Grover developed a quantum algorithm that dramatically sped up the solution to unstructured data searches, running the search in fewer steps than any classical algorithm could.
```
I would remove that sentence since the use of quantum computers for searching in databases is still very speculative and not very likely. The main application of Grover's algorithm is to search for solutions for NP problems and not searching in physical databases. Here a brief discussion from Niels and Chuang's book.

![image](https://user-images.githubusercontent.com/48300381/80979985-5854a600-8e28-11ea-971d-0094f61284dc.png)

Reply:
Done, thanks.